From 57ffde8a2eda4080e413c4a5e89175f1a4e5592e Mon Sep 17 00:00:00 2001
From: Heiko Thiery <heiko.thiery@gmail.com>
Date: Thu, 26 Mar 2020 14:00:35 +0100
Subject: [PATCH] CMakeLists.txt: remove libssh check

add also a patch on the libssh package that fixes the mentioned
regression.

Signed-off-by: Heiko Thiery <heiko.thiery@gmail.com>
---
 CMakeLists.txt | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 92de0d3..f05bac3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -233,10 +233,6 @@ if(ENABLE_SSH)
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DNC_ENABLED_SSH")
     message(STATUS "LibSSH version ${LIBSSH_VERSION} found")
 
-    if(LIBSSH_VERSION VERSION_EQUAL 0.9.3)
-        message(FATAL_ERROR "LIBSSH 0.9.3 includes regression bugs and libnetconf2 will NOT work properly, try to use an older version")
-    endif()
-
     if(LIBSSH_VERSION VERSION_LESS 0.8.0)
         target_link_libraries(netconf2 "-L${LIBSSH_LIBRARY_DIR}" -lssh -lssh_threads -lcrypt)
     else()
-- 
2.20.1

